content/docs/js/Exporter configuration guide for Jaeger.md

```diff
@@ -0,0 +1,68 @@
+# Exporter configuration guide for Jaeger
+
+### Installing Jaeger locally
+
+Download the Jaeger components: https://www.jaegertracing.io/download/(Downloading the docker images should be the better option, (macOS binary wasn’t available (link is broken) when this document was being written))
+Commands for installing the docker image:
+```docker
+$ docker pull jaegertracing/all-in-one:1.17
+```
+Then get Jaeger up and running : (https://www.jaegertracing.io/docs/1.17/getting-started/)
+If you are using Docker, simply use this command:
```

Comment:
I think we should defer to Jaeger's docs for running the docker container or we could fall out of sync if anything changes. A link to their docs with a comment to say "follow these" should be good enough.